drivers/overlay/overlay_cgo.go

```diff
@@ -1,5 +1,5 @@
-//go:build linux && cgo
-// +build linux,cgo
+//go:build linux && cgo && !exclude_disk_quota
+// +build linux,cgo,!exclude_disk_quota
```

Comment:
Rename this file to `overlay_disk_quota.go`. please, now that it is scoped this narrowly.

Reply:
> Rename this file to `overlay_disk_quota.go`. please, now that it is scoped this narrowly.

Okay, thank you. I'll make some changes.